src/components/Toast/__stories__/toast.stories.mdx

```diff
@@ -0,0 +1,157 @@
+
+## Variants
+### Default with button
+<Canvas>
+  <Story name="Default with button">
+    <Toast open className="monday-storybook-toast_wrapper" actions={[{ type: Toast.actionTypes.BUTTON, content: "Button" }]}>General message toast</Toast>
+  </Story>
+</Canvas>
+
+### Toast with link
+<Canvas>
+  <Story name="Toast with link">
+    <Toast open className="monday-storybook-toast_wrapper" actions={[{ type: Toast.actionTypes.LINK, text: "Link to action", href: "https://monday.com" }]}>General message toast</Toast>
+  </Story>
+</Canvas>
+
+### Success message
+Use to providing a feedback loop. For example: Item copied.
+<Canvas>
+  <Story name="Success message">
+    <Toast
+      open 
+      type={Toast.types.POSITIVE} 
+      className="monday-storybook-toast_wrapper" 
+      actions={[{ type: Toast.actionTypes.BUTTON, content: "Undo 5" }]}
+    >
+      Positive message toast
+    </Toast>
+  </Story>
+</Canvas>
+
+### Error message
+Use when something was deleted, a problem has occured, etc.
+<Canvas>
+  <Story name="Error message">
+    <Toast 
+      open 
+      className="monday-storybook-toast_wrapper"
+      actions={[{ type: Toast.actionTypes.BUTTON, content: "Button" }]}
+      type={Toast.types.NEGATIVE}
+    >
+      Negative message toast
+    </Toast>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component: <Toast open className="monday-storybook-toast_wrapper" type={Toast.types.POSITIVE}>Duplicating Board</Toast>,
+        description: "Use only one toast on a screen at a time."
+      },
+      negative: {
+        component:
+          <div className="monday-storybook-toast_inline-wrapper">
+            <Toast 
+              type={Toast.types.NEGATIVE}
+              open
+              hideIcon 
+              closeable={false}
+              className="monday-storybook-toast_wrapper monday-storybook-toast_big"
+              actions={[{ type: Toast.actionTypes.LINK, text: "Try again", href: "https://monday.com" }]}
+            >
+              Couldn’t load board items
+            </Toast>
+            <Toast 
+              type={Toast.types.POSITIVE} 
+              className="monday-storybook-toast_wrapper"
+              hideIcon
+              text="Duplicating Board"
+              open
+            >
+              Duplicating Board
+            </Toast>,
+          </div>,
+        description: <>Don’t place more than one toast or<Link href="/?path=/docs/components-alertbanner--overview">Alert banner</Link>on a screen at a time.</>
+      }
+    },
+    {
+      positive: {
+        component: <Toast open className="monday-storybook-toast_wrapper" type={Toast.types.POSITIVE} actions={[{ type: Toast.actionTypes.BUTTON, content: "Undo" }]}>We successfully archived 1 item</Toast>,
```

Comment:
inside  the do and don't section you can keep the actions value as it is because this code is not  shown to the outside visitors.

Reply:
The actions property has a button with "Undo" content like in the design. We can not write the code otherwise to render the same result. What do you mean by "keep the actions value"?

Reply:
I meant that here you do not need to add the useMemo as in the other stories :)